src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = 40 * days;
+
+  if (days >= 3 && days < 7) {
+    totalCost -= 20;
+  }
+
+  if (days >= 7) {
+    totalCost -= 50;
+  }
```

Comment:
Need to check your code with checklist 
<img width="935" alt="image" src="https://github.com/user-attachments/assets/d9e41323-eae5-48aa-acec-728f74a8bb99" />